### PR TITLE
Changed --cert-path to --tls-cert-path throughout

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/461

@emlin can you please review? 

Do you know when exactly this changed?

### CODE REVIEW

**Comment by stuclem** [comment]
Looks like this was implemented in https://github.com/vmware/vic/issues/4901. @gigawhitlocks can you also please review this PR?

File: docs/user_doc/vic_vsphere_admin/ts_cli_argument_error.md [hunk 1]
```diff
@@ -43,6 +43,6 @@ Option arguments that might require quotation marks include the following:
 - Datastore names and paths in `--image-store` and `--volume-store`
 - Network and port group names in all networking options.
 - Cluster and resource pool names in `--compute-resource`
-- Folder names in the paths for `--cert-path`, `--cert`, `--key`, `--appliance-iso`, and `--bootstrap-iso`
```
Comment (gigawhitlocks) [inline docs/user_doc/vic_vsphere_admin/ts_cli_argument_error.md]: --cert and --key have also changed to become --tls-server-cert and --tls-server-key

File: docs/user_doc/vic_vsphere_admin/ts_cname_mismatch.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The error message means that the existing certificate has a Common Name attribut
   2.  Use the `--cert` and `--key` options to pass the newly generated certificates to `vic-machine create`.
```
Comment (gigawhitlocks) [inline docs/user_doc/vic_vsphere_admin/ts_cname_mismatch.md]: cert & key :\

**Comment by gigawhitlocks** [comment]
All the spots in the docs you found LGTM @stuclem . Just the comment above; I don't know if you want to focus on --cert and --key in another PR.

**Comment by stuclem** [comment]
Thanks @gigawhitlocks. I updated `--cert` and `--key `in this PR.

**Comment by gigawhitlocks** [comment]
LGTM

**Comment by gigawhitlocks** [comment]
LGTM

**Comment by gigawhitlocks** [comment]
LGTM

**Pull request merged**